lib/search-field/index.jsx

```diff
@@ -93,6 +99,7 @@ export class SearchField extends Component {
 }
 
 const mapStateToProps = ({ appState: state }) => ({
+  filter: state.filter,
```

Comment:
ah very good - so this copy replaces the copy passed in as ` query={ filter }` in the parent component - we can remove that property in `app-layout.js`